### ISSUE TITLE
add checkbox to add private note to a thread in customer service

### ISSUE THREAD

**Issue by alfonsolr09** [PrestaShop/PrestaShop#23368]
#### Describe the solution you'd like

I think being able to add a private note for a customer conversation thread would be a good idea.
There are times when customer service is also dealt with by phone or via WhatsApp, I think being able to add a private note to the thread can be a very good idea. That note could be seen by all employees

#### Additional context

![image](https://user-images.githubusercontent.com/62474824/108924440-ecebf700-763a-11eb-8294-d0c891dcf9ac.png)

it would be the same as the one that already exists in the clients tab

![image](https://user-images.githubusercontent.com/62474824/108924905-e3af5a00-763b-11eb-8dbd-c2587eacfcf3.png)

**Comment by hibatallahAouadni** [comment]
Hello @alfonsolr09 

Sorry, but I didn't seem to understand why you need to add a new block (like the attached one below)

![image](https://user-images.githubusercontent.com/23308427/108981190-c787d900-768c-11eb-9005-f99b4b3aecf8.png)

under the message that you sent to your Customers:

![image](https://user-images.githubusercontent.com/23308427/108981350-f1d99680-768c-11eb-8e5a-60dec06b5b11.png)

Could you please provide me an exemple :pray: 
Thanks!

**Comment by alfonsolr09** [comment]
be able to add a private note to a conversation that the customer opened to customer service, i mean

**Comment by alfonsolr09** [comment]
Like in the firts pictures

**Comment by hibatallahAouadni** [comment]
@alfonsolr09 you mean to add a private not for emplyees about the Customer Service default message, is that so?

**Comment by alfonsolr09** [comment]
No, i mean can add a private note about the message that the customer send to me.
Imagine you call to customer to talk about a message that he sent, i can write a private note in the message...

**Comment by hibatallahAouadni** [comment]
Hello @alfonsolr09 

Well that seems a good idea and very helpful to the merchant.
Thanks for your feedback and your suggestion, the Product Team will take a look.
Please stay tuned !

Ping @PrestaShop/product-team , I think it's a good idea to add a private note for the employee (to see) when a customer send message (not quite understandable), wdyt?

**Comment by alfonsolr09** [comment]
I mean, imagine that a client contacts you through the contact form.
later you call him to clarify a problem, but in the company there are several workers.
I think it is a good option to be able to add a private note.
you tell him by phone that you are going to send him another new article.
you add a private note in that conversation: "on 04/10/2021 we spoke with Antonio via phone call, we have agreed to send him a new article".
in order to be able to keep a record of telephone conversations or annotations spoken with the client

**Comment by Hlavtox** [comment]
@alfonsolr09 I know what you mean.

This function is already implemented on order page.

But NOT on the customer threads page.

Also, the option to select predefined message is missing.
